ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -36,7 +36,16 @@ def getRRDTemplateName(self):
         return 'WinService'
 
     def getRRDTemplates(self):
-        return [self.getRRDTemplateByName(self.getRRDTemplateName())]
+        template_name = self.getRRDTemplateByName(self.getRRDTemplateName())
```

Comment:
this should probably be just getRRDTemplateName() instead of getRRDTemplateByName(getRRDTemplateName())